GitVersionExe/ArgumentParser.cs

```diff
@@ -12,7 +12,7 @@ public class ArgumentParser
     {
         static ArgumentParser()
         {
-            var fields = typeof(VariableProvider).GetFields(BindingFlags.Public | BindingFlags.Static);
+            var fields = typeof(VersionVariables).GetProperties();
             VersionParts = fields.Select(x => x.Name.ToLower()).ToArray();
```

Comment:
I see an 'AvailableVariables' property contined in VersionVariables. To access that property I need to instantiate a new VersionVariables instance.